README.md

```diff
@@ -8,26 +8,32 @@
 [![LoC](https://tokei.rs/b1/github/cosmos/gaia)](https://github.com/cosmos/gaia)
 [![GolangCI](https://golangci.com/badges/github.com/cosmos/gaia.svg)](https://golangci.com/r/github.com/cosmos/gaia)
 
-This repository hosts `Gaia`, the first implementation of the Cosmos Hub based on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
 
-**Note**: Requires [Go 1.14+](https://golang.org/dl/)
+## Mainnet Full Node Quick Start
 
-## Cosmos Hub Mainnet
-
-To run a full-node for the mainnet of the Cosmos Hub, first [install `gaiad`](./docs/gaia-tutorials/installation.md), then follow [the guide](./docs/gaia-tutorials/join-mainnet.md).
-
-For status updates and genesis file, see the [launch repo](https://github.com/cosmos/launch).
-
-## Quick Start
+This assumes that you're running Linux or MacOS and have installed [Go 1.14+](https://golang.org/dl/).  It will build and install Gaia, allow you to name your node, add seeds to your config file, start your node and use gaiacli to check the status of your node.  Welcome to the Cosmos!
 
 ```
+git clone -b v2.0.9 https://github.com/cosmos/gaia
+cd gaia
 make install
+gaiad init yournodenamehere
+SEEDS=$(cat seeds); original_string="seeds = \"\""; replace_string="seeds = \"$SEEDS\""; sed -i -e "s/$original_string/$replace_string/g" "$HOME/.gaiad/config/config.toml"
```

Comment:
Just list the seeds in the README file. I think that'll read better and it'll be easier to maintain 👍

Reply:
You're right.  This is simpler, though it does add another step.  

Another reason for the seeds file is that I wanted to put a hundred seeds in there.  

At the same time, I'm sure we can all agree that's some pretty ugly bash and we may be better off without it.  

How about no seeds file, 5 seeds listed right in readme.md?

Reply:
Goal with putting seeds here (in a seeds file or in the readme) is to allow a fresh user to out up a node without navigating away from readme.md

Reply:
Yes, I understand the goal. Just limit the seeds to 10-15 or so 👍

Reply:
OK, I'll make the changes now, thanks.